### PR TITLE
JBPM-9822 : BPMN Editor - Cannot import some processes

### PR DESCRIPTION
**Pull request by inodeman**
Hi @romartin can you review this PR?
**Thank you for submitting this pull request**

**JIRA**: [JBPM-0000](https://issues.redhat.com/browse/JBPM-0000)

**Referenced Pull Requests**:
* paste the link(s) from GitHub here
* link 2
* link 3 etc.

**Business Central**: [WAR file](https://donwnload.here/something)

**VS Code**: [plugin](https://donwnload.here/something)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

**Comment by hasys** [comment]
jenkins do fdb

**Comment by hasys** [comment]
Hi @inodeman, thank you! I was able to upload and open the file.

I tested with WAR from full downstream build. Here it is if it will be removed: [business-central.war](https://drive.google.com/file/d/1-Et8Dk4T08n0zqaXiMO9dLci4vOoYrHJ/view?usp=sharing)

**Review by hasys — APPROVED** [review]

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/converters/customproperties/VariableDeclarationTest.java [hunk 1]
```diff
@@ -83,4 +83,10 @@ public void testToString() {
         comparable = new VariableDeclaration(CONSTRUCTOR_IDENTIFIER, CONSTRUCTOR_TYPE, "");
         assertEquals(comparable.toString(), CONSTRUCTOR_IDENTIFIER + ":" + CONSTRUCTOR_TYPE);
     }
+    
+    @Test
+    public void testToStringNoException() {
```
Comment (LuboTerifaj) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/test/java/org/kie/workbench/common/stunner/bpmn/backend/converters/customproperties/VariableDeclarationTest.java:88]: Hello @inodeman , wouldn't be better to change the name of this test method to something like `testToStringEmptyValue`?

**Review by LuboTerifaj — APPROVED** [review]
Hello @inodeman ,

the issue is fixed in Business central. It is possible to open attached process in jira as well as to create/edit any process with empty MI Data Input/Output variable name and "java.lang.Object" type in Multiple Instance nodes.

If there was any other issue causing that the process couldn't have been opened, please let me know.

Do you plan also open the PR for Kogito?

Thank you!

**Pull request merged**